From bc244bd69fa73b357d84d240da00609cd07f5bbc Mon Sep 17 00:00:00 2001
From: "duncan.macleod" <duncan.macleod@ligo.org>
Date: Mon, 24 Apr 2023 20:54:04 +0100
Subject: [PATCH] remove unnecessary omp link

---
 src/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 3c2b596..458f8c6 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -134,7 +134,6 @@ target_include_directories(libbayeswave PUBLIC
     )
 target_link_libraries(libbayeswave
     m
-    omp
     ${LIBGSL}
     ${LIBFFTW3}
     ${LIBFFTW3F}
-- 
2.40.0

